clients/client-python/gravitino/filesystem/gvfs_config.py

```diff
@@ -35,3 +35,8 @@ class GVFSConfig:
     GVFS_FILESYSTEM_BY_PASS = "gravitino.bypass"
     GVFS_FILESYSTEM_BY_PASS_GCS = "gravitino.bypass.gcs."
     GVFS_FILESYSTEM_KEY_FILE = "service-account-key-path"
+
+    GVFS_FILESYSTEM_BY_PASS_S3 = "gravitino.bypass.s3."
```

Comment:
I was thinking that why do we need a prefix "gravitino.bypass.s3." for gcs and aws configurations? I think the configuration keys mentioned above are important for a client to work, it may not be good to use a bypass prefix, what do you think?

Reply:
OK, let me think a bit.